src/app/lib/device/ext-player.js

```diff
@@ -221,11 +221,12 @@
         folderName = path.resolve(folderName);
         win.info('Scanning: ' + folderName);
         var appIndex = -1;
-        var fileStream = readdirp(folderName, {
+        var fileStream = readdirp({
+            root: folderName,
             depth: 3
         });
         fileStream.on('data', function (d) {
-            var app = d.basename.replace('.app', '').replace('.exe', '').toLowerCase();
+            var app = d.name.replace('.app', '').replace('.exe', '').toLowerCase();
```

Comment:
Please recheck because it cause error on Linux:

![Screenshot from 2020-04-02 19-24-15](https://user-images.githubusercontent.com/11866125/78248836-8d16ca00-7517-11ea-86a1-99d04ebdcd8a.png)

Reply:
also sasme issue with version
pls check you local version, for me yarn install set   "version": "2.2.1"